chance/p5js/chance-praegnanz/index.md

```diff
@@ -1,11 +1,11 @@
 ---  
 layout: p5js
-author: [josues]
-author-url: [https://incom.org/profil/7029]
-title: [Law of Praegnanz and chance]
-tags: [P5.js,[CHANCE], [PROXIMITY], Law of Praegnanz]
-permalink: /[CHANCE]/[chance-praegnanz]/
-summary: [This creates random Praegnanzen between ellipses.]
+author: josues
+author-url: https://incom.org/profil/7029
+title: Law of praegnanz and chance
+tags: P5.js, chance, proximity, Law of praegnanz
+permalink: /chance/chance-praegnanz/
+summary: This creates random Praegnanzen between ellipses.
 ---  
 
 ![](https://raw.githubusercontent.com/josues/gestalten-in-code/master/chance/p5js/chance-praegnanz/out.png)  
```

Comment:
Please no hardcoded urls,  If you want to refer to an image next to yours relativ markdown syntax  


    ![alt text](./out.png)  

But the code from your index.js will be integrated into the final site.